aws-precise-staging-1/main.tf

```diff
@@ -116,7 +117,7 @@ module "aws_asg_com" {
   worker_docker_image_php = "${var.latest_docker_image_php}"
   worker_docker_image_python = "${var.latest_docker_image_python}"
   worker_docker_image_ruby = "${var.latest_docker_image_ruby}"
-  worker_docker_self_image = "quay.io/travisci/worker:v2.5.0-8-g19ea9c2"
+  worker_docker_self_image = "${var.latest_docker_image_worker}"
```

Comment:
That comes from `bin/generate-latest-docker-image-tags`.  Previously, this executable generated variables for all of the `ci-*` stack images.